deployments/scripts/port-forward.sh

```
@@ -4,94 +4,111 @@ set -e
+    GATEWAY=true
+fi
 
-# Check if kubectl is available
+# Check prerequisites
 if ! command -v kubectl &> /dev/null; then
     echo "❌ kubectl is not installed"
     exit 1
 fi
 
-# Check if cluster is running
 if ! kubectl cluster-info --context $CLUSTER_CONTEXT &> /dev/null; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)